blank-canvas/functions.php

```diff
@@ -187,30 +102,6 @@ function blank_canvas_customizer_enqueue() {
 add_action( 'customize_controls_enqueue_scripts', 'blank_canvas_customizer_enqueue' );
 
 /**
- * Customizer additions.
- */
-require get_stylesheet_directory() . '/inc/customizer.php';
-
-/**
- * Adds custom classes to the array of body classes.
- *
- * @param array $classes Classes for the body element.
- * @return array
+ * Override Seedlet settings to be like Blank Canvas.
  */
-function blank_canvas_body_classes( $classes ) {
-
-	if ( false === get_theme_mod( 'show_post_and_page_titles', false ) ) {
-		$classes[] = 'hide-post-and-page-titles';
-	}
-
-	if ( false === get_theme_mod( 'show_site_footer', false ) ) {
-		$classes[] = 'hide-site-footer';
-	}
-
-	if ( false === get_theme_mod( 'show_comments', false ) ) {
-		$classes[] = 'hide-comments';
-	}
-
-	return $classes;
-}
-add_filter( 'body_class', 'blank_canvas_body_classes' );
+require get_template_directory() . '/inc/blank-canvas.php';
```

Comment:
To create a child theme of Seedlet that works like Blank Canvas, we can just include this file.